subsys/bluetooth/host/hci_core.c

```diff
@@ -4771,6 +4794,58 @@ static void le_per_adv_sync_lost(struct net_buf *buf)
 		}
 	}
 }
+
+static void le_past_received(struct net_buf *buf)
+{
+	struct bt_hci_evt_le_past_received *evt =
+		(struct bt_hci_evt_le_past_received *)buf->data;
+	struct bt_le_per_adv_sync_synced_info sync_info;
+	struct bt_le_per_adv_sync_cb *listener;
+	struct bt_le_per_adv_sync *per_adv_sync;
+
+	if (evt->status) {
+		/* No sync created, don't notify app */
+		BT_DBG("PAST receive failed with status 0x%02X", evt->status);
+		return;
+	}
+
+	sync_info.conn = bt_conn_lookup_handle(
+				sys_le16_to_cpu(evt->conn_handle));
+
+	if (!sync_info.conn) {
+		BT_ERR("Could not lookup connection handle from PAST");
+		per_adv_sync_terminate(sys_le16_to_cpu(evt->sync_handle));
+		return;
+	}
+
+	per_adv_sync = per_adv_sync_new();
+	if (!per_adv_sync) {
+		BT_INFO("Could not allocate new PA sync from PAST");
```

Comment:
BT_ERR instead?

Reply:
Hmm, I'm not sure actually. It's not an error, as much as it is just the host that isn't configured to handle more PA syncs.

Reply:
Changed to a BT_WARN.